Jenkinsfile

```diff
@@ -18,7 +18,7 @@
 
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value='pipeline-lib@your_branch') _
+@Library(value='pipeline-lib@soumagne/mercury_libfabric') _
```

Comment:
@phender this is the best and easiest solution I think for now (using the PR https://github.com/daos-stack/pipeline-lib/pull/480), this way there is only minimal changes required to the Jenkinsfiles.

Reply:
Reminder to ourselves that we need to land the pipeline-lib change first

Reply:
Usually we land the pipeline-lib PR (and delete the branch). Then update this daos PR back to the default / uncommented pipeline-lib. Then land this PR if testing is okay

Reply:
well imo if you do that then you are going to break all PRs / CIs until this PR has landed :)

Reply:
but maybe not :)

Reply:
But if this PR lands, then wouldn't in-flight daos PRs still try to install `mercury` and `libfabric`? And then landing the pipeline-lib PR means daos PRs try to ALSO install `mercury-libfabric`?

Reply:
I'm not sure what in-flight really means, for sure PRs will have to be updated / rebased if we merge this PR first. If trying to install `mercury-libfabric` when it does not exist does not create any failures maybe it's fine ? that's why I said "maybe not" in my earlier comment. Either way there might be some potential issue that will resolve on itself once everything lands. At the same time, given the state of our CI I'm not that concerned if I break it for 2 minutes as long as we are aware of it :)

Reply:
So I see two options. Either what you mentioned about leaving the pipeline-lib reference in the landing of this PR, OR something like this:
1. Remove the pipeline-lib reference in this PR.
2. Keep the `daos -> libfabric` dependency for now for compatibility
3. Land this PR, since with (1) and (2) it should still work.
4. Land the pipeline-lib PR
5. New daos PR to remove the `daos -> libfabric` dependency

IMO I prefer this approach so we don't have to leave the pipeline-lib reference in place

Reply:
Then I don't know how to land any of this without breaking CI

Reply:
Well that's what I've just been trying to explain :) I'm glad we're reaching the same conclusion. The real issue is that rpms are globally visible once merged, so because of that, any change of that sort will face the same issues.